migrations/1-3-2_1-3-3.lua

```diff
@@ -0,0 +1,9 @@
+for index, force in pairs(game.forces) do
```

Comment:
`game.reload_script()` missing?
Surround by
```
if recipes["graphite-grinding"] and technologies["carbon-processing"]
..
end
```
?

Reply:
In theory, we're not doing anything special with the rocket-silo that would require the scenario script to be reloaded. And migration scripts are run in order, so these items should exist. Note that none of the migration scripts in base check for nil. There was definitely a bug with the migrations scripts before, I'll try to have a look and see if I can determine the cause.

Reply:
The problem was, that the migration wants to change a recipe, which gots deleted in the current version. When someone adds XM mod to the game, then it runs all migrations, but the access to the deleted recipe fails.
Regarding script reload, i've got so much too learn with modding :)